libs/market-info/src/components/market-info/info-market.tsx

```diff
@@ -218,16 +226,6 @@ export const Info = ({ market }: InfoProps) => {
         />
       ),
     },
-    {
-      title: t('Risk factors'),
```

Comment:
question (??): i don't have any context on this, but is there a reason we don't want to display this anymore?

Reply:
riskFactors are missing in some of the market queries. happy to add this back once things get sorted. in the meantime  I want to remove this if it breaks the whole market-info view. the console will be used for fairground incentives and I don't want to leave this breaking bit here. 

<img width="1677" alt="image" src="https://user-images.githubusercontent.com/16125548/189354172-79944bf8-edec-4b7c-b24e-4544deb04cb6.png">

Reply:
IMO the best approach would be to add riskFactors back after we update the market data providers.